test/WebSites/BasicWebSite/Controllers/ActionConstraints/ConsumesAttribute_MediaTypeSuffix.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using BasicWebSite.Models;
+using Microsoft.AspNetCore.Mvc;
+
+namespace BasicWebSite.Controllers.ActionConstraints
+{
+    [Route("ConsumesAttribute_MediaTypeSuffix/[action]")]
+    public class ConsumesAttribute_MediaTypeSuffix : Controller
+    {
+        [Consumes("application/*+json")]
```

Comment:
Ok I'm confused about this.... why would an action declare that it supports **all json data**?  I'd expect to see a concrete media type here, not a wildcard.